spec/index.html

```diff
@@ -351,6 +352,64 @@ <h3>Blank nodes</h3>
   </section>
 
+  <section id="section-triple-terms">
+    <h3>Triple terms</h3>
+
+    <p>As the basic unit of description in RDF, the triple is a simple
+    abstraction for stating a directed relationship between two resources.
+    Sometimes, we need to describe more detailed circumstances underlying such
+    relationships. While we could describe these using more sophisticated
+    modelling, there is a tension between simple, direct statements and more
+    complex, granular descriptions. This may require a trade-off between
+    directness and completeness.</p>
+
+    <p>To bridge this difference, we can use <a
+    data-cite="RDF12-CONCEPTS#dfn-triple-term">triple terms</a> as
+    <em>references</em> to simple statements. A triple term is a fixed object
+    that identifies a statement through its constituent subject, predicate and
+    object components.</p>
+
+    <p>This enables us to describe more concrete circumstances, such as events,
+    relationship entities or tokens in data sources, <em>in relation to</em>
+    the simple statements. That allows us to keep them as is, rather than
+    requiring a complete remodelling into more detailed structures, for the
+    sake of occasionally adding details.</p>
+
+    <p>Conceptually, we can <em>annotate</em> a simple statement with a
+    reference to something more detailed:</p>
+    <pre class="example nohighlight">
+      &lt;Bob&gt; &lt;is interested in> &lt;the Mona Lisa&gt; <sup><strong>~ &lt;Bob's interest></strong></sup> .
+    </pre>
+
+    <p>This annotation refers to Bob's interest as a concrete circumstance, with
+    a date of origin:</p>
+    <pre class="example nohighlight">
+      &lt;Bob's interest> &lt;is an> &lt;Interest> .
+      &lt;Bob's interest> &lt;since> &lt;4th of October 1998> .
+    </pre>
+
+    <p>Formally, an annotation is formed by two triples: the statement itself
+    and a <a data-cite="RDF12-CONCEPTS#dfn-reifying-triple">reifying
+    triple</a>. This expresses the simple relationship and the reifying relation to that from the circumstance, as a concretization thereof:</p>
+    <pre class="example nohighlight">
+      &lt;Bob&gt; &lt;is interested in> &lt;the Mona Lisa&gt; .
+      &lt;Bob's interest> &lt;is a concretization of> <strong>&laquo;&lpar; <small>&lt;Bob&gt; &lt;is interested in> &lt;the Mona Lisa&gt;</small> &rpar;&raquo;</strong> .
+    </pre>
+
+    <p><a data-cite="RDF12-CONCEPTS#dfn-reifier">Reifiers</a> are subjects of
+    triples relating to triple terms using the <code>rdf:reifies</code>
+    predicate. Triple terms should only appear in the <strong>object
+    position</strong> of such reifying triples.</p>
+
+    <p>Note that triple terms only <em>reference</em> statements. This means
+    that we can describe statements without implying them. This is useful for
+    describing unknown propositions, such as suggestions or disputed claims. We
+    can also annotate statements with multiple reifiers, to describe different
```

Comment:
It would be great if we could elaborate on that aspect, since it outlines the practical benefits of using reifiers.

Reply:
Agree here too; see [051a328](https://github.com/w3c/rdf-primer/pull/16/commits/051a328ed372464eb67dda95657cc44672638ffc).